mesh/accounts/views.py

```diff
@@ -1 +1,105 @@
-from django.http import HttpResponse
\ No newline at end of file
+from django.http import JsonResponse, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.core import serializers
+from django.utils.decorators import method_decorator
+from django.views import View
+
+from .models import Accounts
+from ..utils.validate_data import validate_json_and_required_fields
+import json
+
+
+@method_decorator(csrf_exempt, name='dispatch')
```

Comment:
Why is it csrf exempt?

Reply:
Same with the following

Reply:
from what I learned about it, CSRF from Django helps you protect the website/application or maybe the API from receiving malicious requests or spam requests.